packages/web/src/components/list/MultiDataList.js

```diff
@@ -36,30 +40,50 @@ class MultiDataList extends Component {
 		this.state = {
 			currentValue,
 			searchTerm: '',
+			options: props.data || [],
 		};
+		this.internalComponent = `${props.componentId}__internal`;
 		this.type = 'term';
 		this.locked = false;
 
+		props.addComponent(props.componentId);
 		props.addComponent(props.componentId);
```

Comment:
Shouldn't it be for `this.internalComponent`?

Reply:
Ohh right, fixing it 👍